Source/Widgets/InfoBox/InfoBox.js

```diff
@@ -68,21 +69,56 @@ click: function () { closeClicked.raiseEvent(this); }');
         closeElement.innerHTML = '&times;';
         infoElement.appendChild(closeElement);
 
-        var infoBodyElement = document.createElement('div');
-        infoBodyElement.className = 'cesium-infoBox-body';
-        infoElement.appendChild(infoBodyElement);
-
-        var descriptionElement = document.createElement('div');
-        descriptionElement.className = 'cesium-infoBox-description';
-        descriptionElement.setAttribute('data-bind', '\
-html: descriptionSanitizedHtml,\
-style : { maxHeight : maxHeightOffset(40) }');
-        infoBodyElement.appendChild(descriptionElement);
+        var frame = document.createElement('iframe');
+        frame.className = 'cesium-infoBox-iframe';
+        frame.setAttribute('data-bind', 'style : { maxHeight : maxHeightOffset(40) }');
+        frame.setAttribute('sandbox', 'allow-same-origin allow-popups allow-pointer-lock allow-forms'); // allow-scripts allow-top-navigation
```

Comment:
If that is the case, you could add the attribute right before you access the content, and remove it afterward.  Or, you could switch to an approach using a blob URL where you don't have to directly manipulate the contents of the iframe from outside.